environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -14,6 +14,9 @@
     public static final String NETWORKCIDRS = "The network cidrs for the configured vpc";
     public static final String PRIVATE_SUBNET_CREATION = "A flag to enable or disable the private subnet creation.";
     public static final String SERVICE_ENDPOINT_CREATION = "A flag to enable or disable the service endpoint creation.";
+    public static final String PUBLIC_ENDPOINT_ACCESS_GATEWAY = "A flag to select a public or private endpoint access gateway.";
+    public static final String ENDPOINT_ACCESS_GATEWAY_SUBNET_IDS = "Subnet ids for the endpoint access gateway. (Optional)";
```

Comment:
Can you add more description here, please? What is the purpose of the subnetids?

Reply:
Added more detail. Please check out the updated descriptions and let me know if they're sufficient.